README.md

```diff
@@ -90,192 +97,113 @@ let y = Named["string name"];
 
 ## Enum Declarations
 
-When an `enum` is declared, its members are evaluated and a new _enum constructor function_ is
+When an `enum` is declared, its members are evaluated and a new _enum object_ is
 created:
 
-  - When called, the constructor function coerces its argument to an enum value (similar to the 
-    behavior of `String`, `Number`, and `Boolean`), with precedence given to the earlier 
-    declarations: 
-    ```js
-    enum Numbers { zero, one, two, three, alsoThree = 3 }
-    Numbers(1) === Numbers.one
-    Numbers(3) === Numbers.three // (as opposed to `Numbers.alsoThree`)
-    ```
-  - When `new`-ed, the constructor function returns a wrapper `Object` that boxes the enum value
-    (Similar to the behavior of `String`, `Number`, and `Boolean`). 
-
-An _enum constructor function_ has a number of common methods, described in the [API](#API) section
-below.
-
-Enums can also be created programmatically via the global `Enum.create()` method, also described in
-the [API](#API) section.
+  - An _EnumDeclaration_ may have an `extends` clause whose value must be one of %Number%, 
+    %String%, or %Symbol%, which correspond to a <var>hint</var> of `"number"`, `"string"`,
+    or `"symbol"`, respectively. 
+  - If an _EnumDeclaration_ does not have an `extends` clause, it has a _hint_ of `"number"`. 
+  - The _enum members_ of the declaration are evaluated with the provided <var>hint</var>.
+  - An _EnumDeclaration_ may be decorated, and the decorators may add, remove, or modify _enum members_.
```

Comment:
Should this be in a footnote about cross-cutting concerns, seeings as this part is decorator-proposal-dependent?